package/MDAnalysis/coordinates/PDB.py

```diff
@@ -316,6 +316,11 @@ def __init__(self, filename, **kwargs):
             models.append(0)
         if len(crysts) == len(models):
             offsets = [min(a, b) for a, b in zip(models, crysts)]
+            if len(offsets) == 1:
```

Comment:
If this is a Windows-specific hack, should we be even more specific and say
```python
if len(offsets) == 1 and offsets[0] < 0:
   assert os.name == 'nt', "This should ONLY happen on Windows, see PR #2034"
   # happens with windows_overbyte in some cases
   offsets[0] = 0
```

Btw, you can put both statements in the same `if` because the `and` is short circuited: even `offsets = []` will work because `offsets[0]` is never evaluated.